app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/DetailedPageActivity.java

```diff
@@ -0,0 +1,30 @@
+package com.greenfox.fuchsit.zerdareader.activity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.TextView;
+
+import com.greenfox.fuchsit.zerdareader.R;
+import com.greenfox.fuchsit.zerdareader.model.NewsItem;
+
+/**
+ * Created by regnisalram on 1/30/17.
+ */
+
+public class DetailedPageActivity extends AppCompatActivity {
+
+    TextView article;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.detailed_page);
+
+        Intent i = getIntent();
+        NewsItem newsItem = (NewsItem) i.getSerializableExtra("newsItem");
```

Comment:
@regnisalram dont need intent for anything else, just chain it getIntent().getS...